docs/sources/enterprise/recorded-queries.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Recorded queries"
+description = "Grafana Enterprise recorded queries"
+keywords = ["grafana", "query", "queries", "recorded"]
+weight = 300
++++
+
+# Recorded queries
+
+Recorded queries allow you to see trends over time by taking a snapshot of a data point on a set interval. This can give you insight into historic trends.
+
+For our plugins that do not return time series, it might be useful to plot historical data. For example, you might want to query ServiceNow to see a history of request response times but it can only return current point-in-time metrics.
+
+## How recorded queries work
+
+### Note: An administrator must configure a Prometheus datasource and associate it with a [Remote write target](#remote-write-target) before recorded queries can be used.
+
+Recorded queries only work with backend data source plugins. You can recorded three types of queries:
```

Comment:
will users know how to identify backend data sources, or is that a concept that is only exposed to plugin devs?